### DISCUSSION TITLE
How to get shared Query Id to use with API endpoint api/sqlqueries/{id}

### DISCUSSION THREAD

**Discussion by narrsam** [datalust/seq-tickets#1852] [Help]
I need to get access to the results of a specific saved query using the api endpoint api/sqlqueries/{id}, but I don't know where to get the query's Id from.

**Comment by liammclennan** [comment]
Hi @narrsam 

`GET /api/sqlqueries?shared=true` will give you the set of queries. 

https://docs.datalust.co/docs/server-http-api#apisqlqueries
